### PR TITLE
Adding a spec for testing buttons on the show page for works. 

### PR DESCRIPTION
**Pull request by little9**

### CODE REVIEW

File: spec/features/show_page_buttons_spec.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Generated via
+#  `rails generate hyrax:work Pdf`
```
Comment (no-reply) [inline spec/features/show_page_buttons_spec.rb]: Was this code built by this generator?

Comment (little9) [reply spec/features/show_page_buttons_spec.rb]: Ha! In a sense... I'll remove this comment

File: spec/features/show_page_buttons_spec.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+require 'rails_helper'
+include Warden::Test::Helpers
+
+# NOTE: If you generated more than one work, you have to set "js: true"
+RSpec.feature 'Create a PDF', :clean, js: true do
```
Comment (no-reply) [inline spec/features/show_page_buttons_spec.rb]: Is this the correct description for the test? I'm not certain whether we are testing PDF creation, or the show page buttons.

If the latter, let's create the work with a factory, and avoid the browser interactions for work creation. If the former, we can just rewrite the feature description.

File: spec/features/show_page_buttons_spec.rb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  context 'a logged in admin user' do
+    let(:user) { FactoryGirl.create(:admin) }
+
+    before { login_as user }
+
+    scenario do
```
Comment (no-reply) [inline spec/features/show_page_buttons_spec.rb]: Can we add a description for this `scenario`?

**Comment by little9** [comment]
@tom I removed all the PDF creation clicking around and replaced it with just using the factory and visiting the page directly

**Pull request merged**